.github/workflows/_build_test_upload.yml

```diff
@@ -294,26 +299,26 @@ jobs:
         uses: conda-incubator/setup-miniconda@v2
         with:
           python-version: ${{ matrix.python-version }}
-          activate-environment: conda_build_env
+          activate-environment: ${{ env.CONDA_ENV_PATH }}
```

Comment:
I am not sure about using path here. You might want to add a `CONDA_ENV_NAME` as `conda_build_env`. And, define `CONDA_ENV_PATH` by `"${{ MINICONDA_INSTALL_PATH_MACOS }}/${{ CONDA_ENV_NAME }}"`

Reply:
Technically this should work as defined by their definitions for this particular variable:

```
  activate-environment:
    description:
      "Environment name (or path) to activate on all shells. Default is `test`
      which will be created in `$CONDA/envs/test`. If an empty string is used,
      no environment is activated by default (For `base` activation see the
      `auto-activate-base` option). If the environment does not exist, it will
      be created and activated. If `environment-file` is used and you want that
      to be the environment used, you need to explicitely provide the name of
      that environment on `activate-environment`. If using sh/bash/cmd.exe
      shells please read the IMPORTANT! section on the README.md! to properly
      activate conda environments on these shells."
    required: false
    default: "test"
```

From: https://github.com/conda-incubator/setup-miniconda/blob/main/action.yml#L79-L91